src/main.py

```diff
@@ -0,0 +1,170 @@
+import json
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, List
+
+import questionary
+import yaml
+
+
+@dataclass
+class Questionnaire:
+    name: str
+    questions: List[Dict[str, str]]
+
+
+def load_questionnaires(
+    question_dir: Path,
+) -> List[Questionnaire]:
+    """
+    Given a directory containing json files with questions, produces a lis of Questionnaire
+    objects, one Questionnaire for each source file.
+
+    :param question_dir: Path to a directory expected to contain questionnaires as json files.
+    :return: A list of Questionnaires.
+    :raises: TypeError: If question_dir is not a directory path.
+    :raises: RuntimeError: If an unexpected question format is encountered.
+    """
+
+    if not Path.is_dir(question_dir):
```

Comment:
i would use question_dir.is_dir(). i think this method is available for Path objects